packages/fluent/scss/avatar/_variables.scss

```diff
@@ -41,7 +41,7 @@ $_tc-avatar-matrix: (
     outline: (
         normal: ($white, 100, 100)
     )
-);
+) !default;
```

Comment:
do we really need the default flag on private variables ?

Reply:
Probably not, but it wouldn't hurt.